modules/ducktests/tests/ignitetest/tests/discovery_test.py

```diff
@@ -210,37 +165,75 @@ def __simulate_nodes_failure(self, version, properties, nodes_to_kill=1):
 
         logged_timestamps.sort(reverse=True)
 
-        # Failure detection delay.
-        time_holder = int((time_holder - first_terminated[0]) * 1000)
-        # Failure detection delay by log.
-        by_log = epoch_mills(logged_timestamps[0]) - epoch_mills(first_terminated[1])
+        self.__check_and_store_results(data, int((time_holder - first_terminated[0]) * 1000),
+                                       epoch_mills(logged_timestamps[0]) - epoch_mills(first_terminated[1]))
```

Comment:
how about also provide raw list here? duration1 and duration2.

Reply:
What is "raw list" ? This ticket is mostly to provide loading to node crash emulation and some better test parametrization because many params appeared.

Reply:
Fixed